Library/Formula/pkcs11-helper.rb

```diff
@@ -14,6 +14,9 @@ class Pkcs11Helper < Formula
     sha256 "24bb6ab8aa56792d9b658fa617eef363fc9be794dc3327e5afb88eed30ea8af1" => :mavericks
   end
 
+  option "disable-threading", "Build pkcs11-helper without threading support (useful for proper pkcs11 support in openvpn)"
```

Comment:
do i need to add a caveat section either in openvpn or pkcs11-helper to explicit the existence of this options?